### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adally**
https://github.com/brianfrankcooper/YCSB/issues/1212

The YCSB readme is as the following, which is a little misleading.Because we can't use '-p durability=false' to disable durability. Instead it worked when I used '-p durability=SKIP_WAL'. The readme should be written more clearly.

durability: Whether or not writes should be appended to the WAL. Bypassing the WAL can improve throughput but data cannot be recovered in the event of a crash. The default is true.

### CODE REVIEW

File: core/src/main/java/com/yahoo/ycsb/Client.java [hunk 1]
```diff
@@ -857,7 +857,10 @@ public static void main(String[] args) {
           opcount = Integer.parseInt(props.getProperty(RECORD_COUNT_PROPERTY, DEFAULT_RECORD_COUNT));
         }
       }
-
+      if (threadcount > opcount){
```
Comment (busbey) [inline core/src/main/java/com/yahoo/ycsb/Client.java]: this looks unrelated to the README change. can you break it into a different PR please?

Comment (adally) [reply core/src/main/java/com/yahoo/ycsb/Client.java]: Ok
